docs/content/docs/guide/repositorycrd.md

```diff
@@ -205,3 +205,121 @@ and a pull request event.
 - [Github Documentation for webhook events](https://docs.github.com/webhooks-and-events/webhooks/webhook-events-and-payloads?actionType=auto_merge_disabled#pull_request)
 - [Gitlab Documentation for webhook events](https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html)
 {{< /hint >}}
+
+## Scope GitHub token to a list of private and public repositories within and outside namespaces
+
+The proposal helps user to scope GitHub token to a list of provided repositories which exist in same namespace(By providing configuration at Repository level)
+as well as different namespaces(Global Configuration).
+
+It is useful in a scenario where CI Repos Differ from CD Repos, and
```

Comment:
see my other comments i did this morning on the enhancmeent proposal, please try to phrase the documentation not like a proposal but as documentation, feeel free to get help from openai